### ISSUE TITLE
[TreeChart-TreeChart three layer]: 'Root Node' in 'TreeChart three layer' section is not accessible via keyboard navigation

### ISSUE THREAD

**Issue by AtishayMsft** [microsoft/fluentui#31082]
Test Environment:
OS Build: Windows 11
Version: 24H2 (OS Build 26058.1400)
Browser: Edge-dev
URL: [Fluent UI - Controls - React - TreeChart (microsoft.com)](https://developer.microsoft.com/en-us/fluentui#/controls/web/treechart)
Browser Version: 123.0.2420.10 (Official build) dev (64-bit)
Screen reader: NVDA 2023.3.4(2023.3.4.31265)

Pre-requisites:
Turn on NVDA using 'Ctrl+ alt+ N' keys.
Navigate using keyboard.
Repro Steps: 
Open the above-mentioned [URL](https://developer.microsoft.com/en-us/fluentui#/controls/web/treechart) in edge-dev browser.
'TreeChart' page opens.
Navigate to 'Tree Chart' under 'TreeChart three layer' section using tab key.
Navigate on the 'Root node' node in tree chart using tab key/arrow key ,e.t.c.
Observe the issue.
Actual Result:
In the tree chart under the 'TreeChart three layer' section, the 'Root Node' (Parent) is not accessible using keyboard navigation. Users cannot navigate to the 'Root Node' using keyboard.

Observation:
Issue doesn't reproduce for the Root node under 'TreeChart three layer compact' section.

Expected Result
The 'Root Node' (Parent) in the tree chart should be accessible using keyboard navigation. Users should be able to navigate to the 'Root Node' using keyboard, allowing them to explore the hierarchical structure of the chart.

User Impact: The lack of keyboard accessibility for the 'Root Node' limits the usability of the tree chart for users who rely on keyboard navigation and screen reader users. Users may struggle to navigate the chart and access important information.

https://github.com/microsoft/fluentui/assets/98592573/c76c7af3-6484-4b9c-a897-dfc853d2bd25

**Comment by AtishayMsft** [comment]
FYI @Shubhabrata08
